src/functions-and-arrays.js

```diff
@@ -52,14 +143,46 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arrUni) {
+  if(arrUni.length === 0) {
+    return null
+  }
+  else {
+    let word
+    let uniqueArray = []
+    for(let i = 0; i < arrUni.length; i++) {
+      word = arrUni.splice(i, 1)
```

Comment:
Unecessary to splice the original array, when arrUni[i] inside the for loop would give you the word you need to analyze